src/notes_handlers.py

```diff
@@ -49,6 +53,10 @@ def all_notes(notes:NoteBook) -> str:
     if not notes.data:
         return INFO + " You do not have any notes saved"
     res=""
-    for note in notes.values():
-        res += str(note)+"\n"+ "-"*30 + "\n"
-    return res.strip()
+    for title, note in notes.items():
+        res += "---------------------------------------------------------------------------------------------\n"
+        res += f"title:  |  {title}\n"
+        res += f"body:   |  {note.text}\n"
+        res += f"tags:   |  {', '.join(note.tags)}\n"
+        res += "---------------------------------------------------------------------------------------------\n"
+    return res
```

Comment:
![image](https://github.com/LynxUA/yolo-address-book/assets/38425223/8ff05f89-85b6-41ff-be7d-544a3d7be119)
The long body is not supported

Reply:
let it be 100

Reply:
Do you agree with this format of output all notes? If so, I am going to keep your advice and make changes in my branch. Could you please similary update the output of find-note?

Reply:
In general format is very nice. Regarding long body i would use `from textwrap import wrap` to allow multiline body with width 50-70
Also, could you move this formatting to Note __str__, so I could use it in find-note